saleor/graphql/webhook/subscription_types.py

```diff
@@ -124,24 +124,27 @@ def resolve_address(root, _info: ResolveInfo):
         return address
 
 
-class AddressCreated(ObjectType, AddressBase):
+class AddressCreated(SubscriptionObjectType, AddressBase):
     class Meta:
+        dry_run_model_name = "Address"
```

Comment:
Let's discuss here, a naming for current `dry_run_model_name`

Reply:
My suggestion is `root_type` or `resolver_root_type`.

Reply:
`root_type` is ok, for attaching the model name to the subscription types. But we need something that will tell developers that this flag controls also enabling dry run and trigger. 
So, we should somehow connect `root_type`+ something that will tell developer that setting this value activates the dry run and trigger. In other case, there is a risk, that someone who is not familiar with this future, will treat a missing value for `root_type` as a bug in the code. Setting the comment is a workaround as I would prefer to have the variable name that explains everything.
If there is a possibility that some types will have only dry run or only trigger (probably a valid case for TransactionActionRequest), then maybe we should have a flags there?
```
class Meta:
    root_type = "Address"
    enable_dry_run = True
    enable_test_trigger = False
```
There is more lines but it is suuper clean and readable